net/miniupnpc/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem          1.0
 
 name                miniupnpc
 epoch               2
-version             2.2.3
-revision            1
+version             2.2.4
```

Comment:
does the library bane change with this update? I guess to rephrase the question: why are all the "revbumps" beeded?

Reply:
I can’t test all dependencies, but I got port broken error with at least one and had to rebuild the port. Of course alternatively we can drop revbump and wait until someone complains.